fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreBase.java

```diff
@@ -41,9 +41,16 @@
 public abstract class FDBStoreBase {
     @Nonnull
     protected final FDBRecordContext context;
+
     @Nonnull
     protected final SubspaceProvider subspaceProvider;
 
+    @Nullable
+    private Subspace subspace;
+
+    @Nullable
+    private CompletableFuture<Subspace> subspaceFuture;
```

Comment:
I think we generally don't put blank lines between member variable definitions, though I'm not sure we're super consistent one way or the other.